fast_reload_db.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+rm project.db
+./manage.py clear_index --noinput
+cp project.db.empty project.db
```

Comment:
You could have `./manage.py flush` here

Reply:
you can also rename `fast_reload_db.sh` to `clear_data.sh`

Reply:
'./manage.py flush' will not have the same behaviour I intended. 

From the documentation of flush:
"Removes ALL DATA from the database, including data added during migrations.
Does not achieve a "fresh install" state."

Reply:
ok